### ISSUE TITLE
Clarification on spatial scale used to generate latitude/longitude-based species lists?

### ISSUE THREAD

**Issue by cbalantic** [kahst/BirdNET-Analyzer#33]
Hi BirdNET team,

Is it possible to add a brief clarification to the ReadME about the spatial scale/model used to filter the species list based on lat-long?

I started noticing some slight differences in the number of species used -- for example, two sites located about 200m away from one another where one "Species list contains 137 species" and the other "Species list contains 138 species." In retrospect, I realized this was due to different weeks of the year for the recordings I was looking at, rather than the small chance that both sites fell in different grid cells in the underlying model. Nevertheless, it did cause me to wonder about what spatial scale is used in the underlying model to select the species list (i.e., 10km, 1km, [eBird species list grid cells](https://ebird.org/science/status-and-trends/faq#range1)?). Are you able to clarify?

Thank you for this incredible tool!

**Comment by kahst** [comment]
Sure, we used 0.5° grid cells (i.e., 0.5° increments of lat and lon) to parse the eBird API for observation data. We then trained the metadata model using these data with random lat/lon pairs so that the model learns to transition between grid cells. Ideally, the metadata model can now generate a species list that matches the eBird observational data. However, eBird does use different grid sizes internally when a location does have an insufficient amount of observations. So by design, grid cells do overlap and thus, that might be reflected in the metadata model. But we did some verification on the metadata species predictions and they seem to be fairly accurate. A bit of uncertainty is still desirable and we enforced this through data augmentation, because species distribution is not always clear-cut. Due to this, you might find some species predicted at locations where they rarely occur.
